packages/amazon-sumerian-hosts-three/README.md

```diff
@@ -0,0 +1,1000 @@
+# Amazon Sumerian Hosts
+
+For a general introduction to Amazon Sumerian Hosts as well as other example implementations of utilizing the core package, refer to the [primary repository](https://github.com/aws-samples/amazon-sumerian-hosts)
+
+Amazon Sumerian Hosts is an experimental open source project that aims to make it easy to create interactive animated 3D characters that can be rendered on the Web and leverage AWS Services such as [Amazon Polly](https://aws.amazon.com/polly/). This example package includes everything you need to get started with Amazon Sumerian Hosts utilizing [three.js](https://threejs.org/). 
+
+See the [Getting Started](#Getting-Started) guide for a walkthrough using this method and the [API Documentation](https://aws-samples.github.io/amazon-sumerian-hosts/) for more detailed information on the classes and methods available. Amazon Sumerian Hosts is a published [npm](https://www.npmjs.com/) package, so alternatively you can install in an existing Node.js project by running `npm install --save-dev @amazon-sumerian-hosts/three`. If you'd like to pull the gitub repository and create your own build, see [Building the Repository](https://github.com/aws-samples/amazon-sumerian-hosts/blob/mainline/README.md#building-the-repository) for prerequisites and instructions on how to do that.
+
+## License
+
+This library is licensed under the MIT-0 License. See the [LICENSE](LICENSE) file. The assets within examples are licensed under the CC-BY-4.0 License. See the [LICENSE](examples/assets/LICENSE) file.
+<br/><br/>
+
+# [Getting Started](#Getting-Started)
+
+This guide steps through the creation of [examples/three.html](examples/three.html), resulting in an interactive 3D character that animates and speaks the text you input into the textarea, rendered in the Web rendering engine of your choice.
```

Comment:
I have the same request as I did with the Bablyon README. "steps through the creation" is a little confusing to me. It makes it sound like we're asking the user to write the code themselves. Also, some of the other wording (e.g. "rendering engine of your choice") isn't appropriate. I suggest changing this to 

> This guide steps you through the code of the `examples/three.html` file, a working example demonstrating the features of the Sumerian Host characters running in three.js